docs/examples/DataSet/The-Experiment-Container.ipynb

```diff
@@ -159,115 +177,375 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "run_c #3@C:\\Users\\a-fbonabi\\temp_work\\exp_container_tutorial.db\n",
+       "---------------------------------------------------------------"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_data_set('run_c')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's check the content of this experiment:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "second_exp#slightly_newer_sample#2@C:\\Users\\a-halakh\\Documents\\Microsoft\\QcodesHarshit\\docs\\examples\\DataSet\\exp_container_tutorial.db\n",
-      "--------------------------------------------------------------------------------------------------------------------------------------\n",
-      "1-a_good_run-1--0\n",
-      "2-a_nother_run-2--0\n"
+      "second_exp#slightly_newer_sample#2@C:\\Users\\a-fbonabi\\temp_work\\exp_container_tutorial.db\n",
+      "-----------------------------------------------------------------------------------------\n",
+      "2-run_b-1--0\n",
+      "3-run_c-2--0\n"
      ]
     }
    ],
    "source": [
-    "# Printing the experiment will reveal the runs\n",
-    "print(exp_2)"
+    "print(exp2)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In a similar way, we may of course add runs to the other two experiments.\n",
-    "\n",
-    "Now, the alert reader will have noticed that `exp_id` is a keyword argument in `new_data_set`. What is the default then? -- The default is that the run is added to the experiment with the **highest** `exp_id` in the database."
+    "We can also explicitly use `exp_id` in creating runs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "first_run #4@C:\\Users\\a-fbonabi\\temp_work\\exp_container_tutorial.db\n",
+       "-------------------------------------------------------------------"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_data_set('first_run', exp_id=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, the alert reader will have noticed that `exp_id` is a keyword argument in QCoDeS runs. What is the default then? -- By default, the maximum `exp_id` in the database is the default value, and the experiment with that `exp_id` will be used in QCoDeS runs. However, if the user loads or creates an experiment in the Python session, that experiment will be activated and used in runs."
```

Comment:
let's use the word DataSet instead of "run"? i think it's inconsistent if all the methods/functions/classes talk about datasets while the notebook text is using "run"